frontend/package.json

```diff
@@ -54,6 +54,7 @@
     "redux-thunk": "^2.3.0",
     "regenerator-runtime": "^0.13.3",
     "reselect": "^4.0.0",
+    "save": "^2.4.0",
     "string-similarity": "^3.0.0",
```

Comment:
@Jsyro did you install this? wondering if it was it an accidental install on my part...

Reply:
100% confidant that Jason didn't add this unless also by accident, so probably good to remove.